README.md

```diff
@@ -1,13 +1,17 @@
 # 📦✨  wasm-pack
-> pack up the wasm and publish it to npm!
+> your favorite rust -> wasm workflow tool!
 
 [![Build Status](https://travis-ci.org/ashleygwilliams/wasm-pack.svg?branch=master)](https://travis-ci.org/ashleygwilliams/wasm-pack)
 [![Build status](https://ci.appveyor.com/api/projects/status/7jjuo5wewu9lyyfi?svg=true)](https://ci.appveyor.com/project/ashleygwilliams/wasm-pack)
 
-the goal of this project is to create a portable command line tool
-for publishing compiled wasm projects to the npm registry for the consumption
-of js devs using the npm CLI, yarn, or any other CLI tool that interfaces
-with the npm registry.
+this tool seeks to be a one-stop shop for building and working with rust-
+generated webassembly that you would like to interop with JavaScript, in the
+browser or with Node.js. `wasm-pack` helps you build and publish rust-generated
+web assembly to the npm registry to be used alongside any other javascript
+package in workflows that you already use, such as [webpack] or [greenkeeper].
```

Comment:
Maybe mention other bundlers like Browserify (not sure about parcel), but it'll help show we're not partial to any one bundler

Reply:
do we know if those work yet? i'm super down to list them but i dont want to promise something that doesn't exist yet. we might want a "can i use?" at some point for bundlers

Reply:
Well if it's a JS package we're making my understanding is it just should? That was what I thought. Maybe @alexcrichton has a better understanding here. If not then we can expand it later.

Reply:
there are some bugs/blockers in other bundlers last i checked, but that may have changed!

Reply:
Cool well lets just table this comment for now. We can look it up later and update this as necessary

Reply:
May be linking [here](https://github.com/rustwasm/team/blob/master/goals/bundler-integration.md#details) would help